src/main/java/Deadline.java

```diff
@@ -0,0 +1,37 @@
+public class Deadline extends Task {
+
+    private String time;
+
+    /**
+     *  This method updates the action and time accordingly for Deadline.
+     * @param description This is the action to be taken.
+     * @param time This is the time for which that action has to happen.
+     */
+    public Deadline(String description, String time) {
+        super(description);
+        this.time = time;
+    }
+
+    @Override
+    /*Override the toString method for Deadline*/
```

Comment:
This comment is not really necessary, but even if you want to put it, it should conform to the coding style:
should have 
/**
 * ...
 */
even if it is only a one liner